catalog-rest-service/src/main/java/org/openmetadata/catalog/security/DefaultAuthorizer.java

```diff
@@ -123,6 +128,20 @@ public boolean hasPermissions(AuthenticationContext ctx, EntityReference owner)
     }
   }
 
+  @Override
+  public boolean hasPermissions(
```

Comment:
TODO: Need to wire this up to some of the Resource APIs and write tests around authorization of Metadata Operation changes

Reply:
Do this in next PR